src/qibolab/platforms/abstract.py

```diff
@@ -195,53 +176,120 @@ def update(self, updates: dict):
 
         for par, values in updates.items():
             for qubit, value in values.items():
+                log.info(f"qubit {qubit} - param {par} - values {value}")
```

Comment:
This line could be removed I think.
Or we could write a proper log.

Reply:
Removed. I agree. I was using it only to debug.